README.md

```diff
@@ -9,6 +9,43 @@ Supports:
 
 ## Installation
 
+### Using the Swift Package Manager
+
+To integrate Raygun into an SPM managed application, update your Package.swift file to include raygun4apple as a dependency. 
+
+Here is an example of a macOS CLI application which uses raygun4apple as a dependency
```

Comment:
Colon at the end of this sentence?

Reply:
Here is an example of a macOS CLI application which uses raygun4apple as a dependency: